### PR TITLE
Scale virt-launcher to pick up config changes during tests

### PR DESCRIPTION
**Pull request by rmohr**
**What this PR does / why we need it**:

* The config changes are only half picked-up because the functests only scale virt-api and virt-controller. virt-handler needs to be scaled too.
* Don't deploy the test-infra via the test-binary. The test-infra is also needed during development. The test-binary deploys it for the usual development-flow too late and also does not keep the test-infra.

**Which issue(s) this PR fixes** *(optional, in `fixes #<issue number>(, fixes #<issue_number>, ...)` format, will close the issue(s) when PR gets merged)*:
Alternative solution to #1869.

**Special notes for your reviewer**:

**Release note**:

```release-note
NONE
```

### CODE REVIEW

**Comment by rmohr** [comment]
@lukas-bednar @slintes had to update it again. See the updated commit message. I think we need to still deploy the test-infra in our scripts, because we also want the test infra to be available during development.

**Comment by rmohr** [comment]
ci test please

File: cluster/deploy.sh [hunk 1]
```diff
@@ -25,19 +25,32 @@ source hack/config.sh
...
         if [[ $manifest =~ .*demo.* ]]; then
             continue
         fi
-        _kubectl create -f $manifest
+        _kubectl apply -f $manifest
     done
 fi
 
 if [[ "$KUBEVIRT_PROVIDER" =~ os-* ]]; then
+    _kubectl create -f ${MANIFESTS_OUT_DIR}/testing/ocp
```
Comment (lukas-bednar) [inline cluster/deploy.sh]: you will need to move that scc manifest into this directory.

this change will affect that *-for-k8s.yaml & *-for-ocp.yaml fnmatch which I use to read appropriate manifests for relevant cluster.

**Comment by rmohr** [comment]
Seems like some lanes had temporary connectivity issues:
```
local-volume-provisioner-slrxd   0/1       ImagePullBackOff   0         24m
```
For reference:

http://jenkins.ovirt.org/job/kubevirt_kubevirt_standard-check-pr/2984/artifact/ci_build_summary.html

**Comment by rmohr** [comment]
ci test please

**Pull request merged**